packages/components/socialFeed/Map/Map.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+import { LeafletView } from "react-native-leaflet-view";
+
+import { MapProps } from "@/components/socialFeed/Map/Map.types";
+import { DEFAULT_MAP_POSITION, MAP_LAYER_URL } from "@/utils/feed/map";
+
+export const Map: FC<MapProps> = ({
+  creatingPostLocation,
+  locationToCenter = DEFAULT_MAP_POSITION,
+}) => {
+  return (
+    <LeafletView
+      renderLoading={() => <></>}
+      zoom={12}
+      mapCenterPosition={creatingPostLocation || locationToCenter}
+      mapLayers={[
+        {
+          url: MAP_LAYER_URL,
+        },
+      ]}
+      mapMarkers={[
+        {
+          position: creatingPostLocation || locationToCenter,
+          icon: "https://i.ibb.co/gSnJ70P/location.png", //load image from web; not work with local image
```

Comment:
Is that normal I have image not found with this link ? 
can we ensure that this link always exists ? if not then I think an ipfs link or our host would be better

Reply:
I removed this wrong icon url.
`Map.tsx` is the component loaded on mobile. We'll do the mobile version of a lot of features later